.betterer.results

```diff
@@ -7229,6 +7229,10 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "3"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "4"]
     ],
+    "public/app/plugins/datasource/opentsdb/components/MetricSection.tsx:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
+    ],
```

Comment:
Please do not add new exceptions into the betterer file. Our goal is to remove this tool completely, which we cannot do if we continue to grow it.

Please either type your code correctly, or use `// eslint-ignore-next-line` if following the lint rule is absolutely not possible.

Reply:
Thank you Josh, sorry about that! I got a result from betterer where is said I had used 'any' at the top of a file and I thought that was a false positive. It was not, I found it and used the correct type.